### ISSUE TITLE
What happened to Helper.log?

### ISSUE THREAD

**Issue by postmechanical** [fastlane/fastlane#4827]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

##### Complete output when running fastlane, including the stack trace and command used

```
[15:20:47]: fastlane finished with errors
/Users/travis/build/lonelyplanet/lpm-flagship-ios/fastlane/actions/git_unshallow.rb:11:in `run': [!] undefined method `log' for FastlaneCore::Helper:Module (NoMethodError)
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:177:in `block (2 levels) in execute_action'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/actions/actions_helper.rb:35:in `execute_action'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:162:in `block in execute_action'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:161:in `chdir'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:161:in `execute_action'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:109:in `trigger_action_by_name'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/fast_file.rb:140:in `method_missing'
	from Fastfile:25:in `block (2 levels) in parsing_binding'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:191:in `call'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:191:in `execute_flow_block'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:47:in `block in execute'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:46:in `chdir'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/runner.rb:46:in `execute'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/lane_manager.rb:46:in `cruise_lane'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/commands_generator.rb:42:in `block (2 levels) in run'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane_core-0.44.1/lib/fastlane_core/ui/fastlane_runner.rb:26:in `run!'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/commands_generator.rb:159:in `run'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/commands_generator.rb:13:in `start'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/lib/fastlane/cli_tools_distributor.rb:47:in `take_off'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.90.0/bin/fastlane:5:in `<top (required)>'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/bin/fastlane:23:in `load'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/bin/fastlane:23:in `<main>'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/bin/ruby_executable_hooks:15:in `eval'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/bin/ruby_executable_hooks:15:in `<main>'
```

### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Fastfile:
```ruby
```

Not relevant.

### Environment

fastlane version (run `fastlane -v`): 1.90.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? No

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? No

**Comment by asfalcone** [comment]
`Helper.log` was [removed here](https://github.com/fastlane/fastlane/pull/4709/files#diff-6c7c4ae2979901f56b7f8a626b9ced92L8).

We're reviewing this to see if we need to revert that code or recommend a new way of logging. Hang tight!

**Comment by postmechanical** [comment]
Lack of info in release notes for such a breaking change is really frustrating. The rate at which changes to fastlane cause build failures is reaching a tipping point where the usefulness of the tool is undermined.

**Comment by asfalcone** [comment]
@postmechanical We're going to revert change and I'll let you know as soon as we ship a release. Really sorry about this!

**Comment by asfalcone** [comment]
@postmechanical This has been fixed and released. `gem update fastlane_core` and you should be good to go! Thanks for reporting and patiently waiting on the fix  👍

**Comment by KrauseFx** [comment]
If you're using bundler you can use `bundle update` to updated your dependencies. 

Also from my side, I'm really sorry for introducing this change, and even worse, not even mentioning it in the release notes. I just didn't think of external actions when introducing this change. There is good excuse for doing so, sorry this happened - please accept my apology. I'll be much more careful with future changes and promise I'll improve 👍
